diff --git a/busybox_sorted_single_03.uvl b/busybox_sorted_single_04.uvl
index 0a2503d..0161aad 100644
--- a/busybox_sorted_single_03.uvl
+++ b/busybox_sorted_single_04.uvl
@@ -111,6 +111,7 @@ features
 			CONFIG_FEATURE_FIND_EXEC
 			CONFIG_FEATURE_FIND_GROUP
 			CONFIG_FEATURE_FIND_INUM
+			CONFIG_FEATURE_FIND_MAXDEPTH
 			CONFIG_FEATURE_FIND_MMIN
 			CONFIG_FEATURE_FIND_MTIME
 			CONFIG_FEATURE_FIND_NEWER

