diff --git a/busybox_sorted_single_04.uvl b/busybox_sorted_single_05.uvl
index 0161aad..9d36e97 100644
--- a/busybox_sorted_single_04.uvl
+++ b/busybox_sorted_single_05.uvl
@@ -121,6 +121,7 @@ features
 			CONFIG_FEATURE_FIND_PERM
 			CONFIG_FEATURE_FIND_PRINT0
 			CONFIG_FEATURE_FIND_PRUNE
+			CONFIG_FEATURE_FIND_REGEX
 			CONFIG_FEATURE_FIND_SIZE
 			CONFIG_FEATURE_FIND_TYPE
 			CONFIG_FEATURE_FIND_USER

