diff --git a/busybox_sorted_single_32.uvl b/busybox_sorted_single_33.uvl
index 6b25b87..00c49d4 100644
--- a/busybox_sorted_single_32.uvl
+++ b/busybox_sorted_single_33.uvl
@@ -247,6 +247,7 @@ features
 			CONFIG_FEATURE_RUNCON_LONG_OPTIONS
 			CONFIG_FEATURE_SECURETTY
 			CONFIG_FEATURE_SETCONSOLE_LONG_OPTIONS
+			CONFIG_FEATURE_SETFILES_CHECK_OPTION
 			CONFIG_FEATURE_SGI_LABEL
 			CONFIG_FEATURE_SH_EXTRA_QUIET
 			CONFIG_FEATURE_SH_IS_ASH

