diff --git a/automotive02_sorted_single_30.uvl b/automotive02_sorted_single_31.uvl
index 2974c6d..4438a7f 100644
--- a/automotive02_sorted_single_30.uvl
+++ b/automotive02_sorted_single_31.uvl
@@ -970,6 +970,7 @@ features
 											I_189205934__F_50C1B9F563B7
 											I_189205934__F_558981D7E2F2
 											I_189205934__F_5EAF2384F11A
+											I_189205934__F_88C0EBD8EFA5
 											I_189205934__F_949CA0B755E9
 											I_189205934__F_A66FE24B20D6
 											I_189205934__F_A6CADF6D797C

