diff --git a/automotive02_sorted_single_73.uvl b/automotive02_sorted_single_74.uvl
index 425f7b4..7484f65 100644
--- a/automotive02_sorted_single_73.uvl
+++ b/automotive02_sorted_single_74.uvl
@@ -3121,6 +3121,7 @@ features
 									F_AC60E106EC44
 							"N_2056103036__F_360D6496FCD3"
 								optional
+									F_3CB5AE78E87F
 									N_2056103036__F_053DB81D3402
 									N_2056103036__F_715BED3FC5A3
 									N_2056103036__F_BD2AF3EE75CA

