diff --git a/automotive02_sorted_single_33.uvl b/automotive02_sorted_single_34.uvl
index 857ffaa..736218c 100644
--- a/automotive02_sorted_single_33.uvl
+++ b/automotive02_sorted_single_34.uvl
@@ -1244,6 +1244,7 @@ features
 									F_CB57FD4FD2D2
 									F_D059354143E6
 									F_D06F3DEBB846
+									F_D0C443A40FFD
 									F_D0D20E6E701A
 									F_D62965A0E5DA
 									F_D777908F15E2

