diff --git a/automotive02_sorted_single_74.uvl b/automotive02_sorted_single_75.uvl
index 7484f65..2a2b096 100644
--- a/automotive02_sorted_single_74.uvl
+++ b/automotive02_sorted_single_75.uvl
@@ -3128,6 +3128,7 @@ features
 									N_2056103036__F_F63979FF259B
 							"N_2056103036__F_38407F2D6979"
 								optional
+									F_0361034F7AE2
 									N_2056103036__F_5C5FDA183ADE
 							F_09A009BB31E8
 							F_0E13498C0222

