diff --git a/automotive02_sorted_single_57.uvl b/automotive02_sorted_single_58.uvl
index 8fac557..1539154 100644
--- a/automotive02_sorted_single_57.uvl
+++ b/automotive02_sorted_single_58.uvl
@@ -2753,6 +2753,7 @@ features
 									F_1DC4CF347EA0
 									F_284256528569
 									F_28CBC87F8364
+									F_3EB4DD757423
 									F_62D64668E900
 									F_8ED933945E0A
 									F_BADA3349D6F0

