diff --git a/automotive02_sorted_single_91.uvl b/automotive02_sorted_single_92.uvl
index b4f2ec1..8b2b1ab 100644
--- a/automotive02_sorted_single_91.uvl
+++ b/automotive02_sorted_single_92.uvl
@@ -3360,9 +3360,6 @@ features
 							F_78444AD28000
 							F_857BAA3AA7D1
 							F_88EA87962355
-							F_90CCCD098AA6
-							F_976B6B409804
-							F_A117CF33BF99
 							F_ACA82FF661F1
 							F_AFC097A506F3
 							F_B2D554FC2D33

