diff --git a/automotive02_sorted_single_04.uvl b/automotive02_sorted_single_05.uvl
index 2818f6e..285e215 100644
--- a/automotive02_sorted_single_04.uvl
+++ b/automotive02_sorted_single_05.uvl
@@ -38,53 +38,6 @@ features
 								alternative
 									F_44C5F3D79493
 									F_E0241971AF07
-							"F_32C254010D41"
-								alternative
-									F_1B827832FB4C
-									F_4FFA588DA0A4
-									F_B8B3E9EF0DFF
-							"F_32FAA9606809"
-								alternative
-									F_2873704C772B
-									F_84FD31A6A208
-									F_8A4A3330DCAE
-									F_B9B0C039C5FA
-							"F_36F89A3D7AD5"
-								alternative
-									F_6702FBE6AF4A
-									F_BC26389FF0F1
-							"F_3789CE8AEB20"
-								or
-									F_0978DE94C670
-									F_235BD743A458
-									F_2E30929AC742
-									F_3273501277B5
-									F_3EA7DBC617D1
-									F_40698B9CC185
-									F_5C7CA2B942CD
-									F_61BDC80A74D0
-									F_6E737CC0E884
-									F_88EF1BB15978
-									F_921B7DA7E671
-									F_C2B1FFCCDAFF
-									F_C648B0CFB5D9
-									F_D2C30ECA6D0E
-							"F_39B7B90A3EF6"
-								optional
-									F_0D898DCED33A
-									F_6635C90E8BCF
-									F_714B32917AB1
-									F_951079130B5A
-									F_BCC115CCE7A2
-									F_ECEC6E9A13BC
-							"F_3AB7EF4398FA"
-								alternative
-									F_064080E1B52E
-									F_71C26DAEF5B2
-							"F_3ABB1F1894AD"
-								alternative
-									F_8750C1B9DEAA
-									F_9B73B1927DB3
 							"F_45DF2BE75B2C"
 								alternative
 									F_51DCD7ED3F8D

