diff --git a/automotive02_sorted_single_69.uvl b/automotive02_sorted_single_70.uvl
index aff4f16..1e7be3b 100644
--- a/automotive02_sorted_single_69.uvl
+++ b/automotive02_sorted_single_70.uvl
@@ -3016,6 +3016,7 @@ features
 									F_0A531AEC540F
 									F_12B2937AA236
 									F_156E67B825AA
+									F_1CAEDA2EEF11
 									F_1E6FAC64C392
 									F_2F7090DA0FF1
 									F_93990CB3595E

