diff --git a/automotive02_sorted_single_75.uvl b/automotive02_sorted_single_76.uvl
index 2a2b096..9d2a8a2 100644
--- a/automotive02_sorted_single_75.uvl
+++ b/automotive02_sorted_single_76.uvl
@@ -3131,7 +3131,6 @@ features
 									F_0361034F7AE2
 									N_2056103036__F_5C5FDA183ADE
 							F_09A009BB31E8
-							F_0E13498C0222
 							F_1A805DA3564D
 							F_24632921D199
 							F_29FDBC0F24D1

