diff --git a/automotive02_sorted_single_24.uvl b/automotive02_sorted_single_25.uvl
index 9f3281b..3f8e3aa 100644
--- a/automotive02_sorted_single_24.uvl
+++ b/automotive02_sorted_single_25.uvl
@@ -882,6 +882,7 @@ features
 											I_4139709915__F_05F7F3819D55
 											I_4139709915__F_0D969C693246
 											I_4139709915__F_151567FBE230
+											I_4139709915__F_17B01B151064
 											I_4139709915__F_21683A03116E
 											I_4139709915__F_2F77A92353D7
 											I_4139709915__F_3AC122BB9910

