diff --git a/automotive02_sorted_single_12.uvl b/automotive02_sorted_single_13.uvl
index 1a83938..24bfb6e 100644
--- a/automotive02_sorted_single_12.uvl
+++ b/automotive02_sorted_single_13.uvl
@@ -256,6 +256,57 @@ features
 									F_CAD4DBAB0178
 							"F_87DD8E8C2B99"
 								alternative
+									"F_4C33E5FDD98F"
+										mandatory
+											"F_28672FAC534F"
+												alternative
+													F_0FF4E50AB5A4
+													F_191859F27323
+													F_1BEEA28F4E03
+													F_9A5EC5CEF22C
+													F_BA6F080A4022
+													F_F0ABBBEE812E
+											"F_32FAA9606809"
+												alternative
+													F_2873704C772B
+													F_84FD31A6A208
+													F_8A4A3330DCAE
+													F_B9B0C039C5FA
+											"F_36F89A3D7AD5"
+												alternative
+													F_6702FBE6AF4A
+													F_BC26389FF0F1
+											"F_9DBED97E3EFE"
+												alternative
+													F_13C2103B3DE1
+													F_4F7D59B1A297
+											"F_F61D7DF4F3BD"
+												alternative
+													F_7FF179F5D449
+													F_FFC82B5D98EB
+											"I_2063493806__F_154C88655C50"
+												alternative
+													I_2063493806__F_825957E77D7E
+													I_2063493806__F_CBD822BE4A9E
+													I_2063493806__F_CBFD51FEB81A
+													I_2063493806__F_F69E7F706F2E
+										optional
+											"F_C16BFEF10D54"
+												alternative
+													F_0FD10D26F7B1
+													F_76A7AD4A7AED
+											F_02F738740212
+											F_14AE801EFA46
+											F_372BCED4FAC5
+											F_52649EEC5F86
+											F_8ABA34AC977C
+											F_9320A0C39F50
+											F_946EBE87B97C
+											F_A3C5A584B271
+											F_A9BC94361248
+											F_CAF68868D4B3
+											F_CF8F0096C625
+											F_ED163BF04379
 									F_1C6783CDF11E
 									F_4C33E5FDD98F
 							"F_88AADDE4C705"

