diff --git a/automotive02_sorted_single_80.uvl b/automotive02_sorted_single_81.uvl
index 093d37c..e9f01c5 100644
--- a/automotive02_sorted_single_80.uvl
+++ b/automotive02_sorted_single_81.uvl
@@ -3201,6 +3201,92 @@ features
 									F_9AA70024D264
 							"F_80C554B028DE"
 								alternative
+									"F_EE9476FFE732"
+										mandatory
+											"F_0D863355DCCB"
+												mandatory
+													"F_0F56B420D903"
+														alternative
+															F_8A678EEDBBF8
+															F_E16E36F43A2D
+															F_F32BC957CC54
+													"F_43BEAA4DE1C7"
+														alternative
+															F_69797AEBA8BE
+															F_BED8B2E508C8
+													"F_6029AB7E62B8"
+														alternative
+															F_28D363D19177
+															F_6FD088FDB5F8
+															F_AB01C1EEE6C8
+													"F_7385E1C0F98A"
+														alternative
+															F_64DB354CF861
+															F_E179154FB7C9
+															F_F05E7D749232
+													"F_7DC7CA8A4DA6"
+														alternative
+															F_1A26AF716319
+															F_A424AC25A4A6
+															F_DCD816E8D37F
+													"F_9DA86B738946"
+														alternative
+															F_57B7FC6A45B2
+															F_E5E7EF9557AC
+															F_F1FBA3F84A4E
+													"F_9FD1D889BBCA"
+														alternative
+															F_1657A8B3E71A
+															F_6EEF04D18757
+															F_B0EB1862CB70
+													"F_A7ED3B30AFE4"
+														alternative
+															F_0E3EF5734ABE
+															F_CBCB55BC4E85
+															F_D4A9E12927D3
+													"F_AD17FA95298D"
+														alternative
+															F_D3C37AECE9CA
+															F_DD3B870329AC
+													"F_BA3C2B12FAA8"
+														alternative
+															F_C83981E50D1E
+															F_D7FF4FD816D0
+												optional
+													F_CE7405803912
+													F_E35B3E088311
+											"F_466576092406"
+												alternative
+													"F_D8A356C2678B"
+														optional
+															F_3553C05C14E3
+															F_90CCCD098AA6
+															F_D37C91A69D94
+													F_A859E1007485
+											"F_69B971E4D5DC"
+												alternative
+													F_3B843A5BA7AF
+													F_42A527DB2CC6
+											"F_8E0692694689"
+												alternative
+													F_8E882668C971
+													F_B510F230D08E
+											"F_D301A881B9D5"
+												alternative
+													F_2E7C625562B3
+													F_7F86D1D33DE9
+											"F_ED39CF9B120E"
+												alternative
+													F_6EAE78C49F48
+													F_7D7D7BDEC9CF
+										optional
+											"F_491B5B403498"
+												or
+													F_2445132C54F1
+													F_CAA23D469716
+											F_300290481B1A
+											F_65C1805B4944
+											F_741051FEE4BC
 									F_789AA20B61FF
 									F_EE9476FFE732
 							"F_A58156FCE84F"

