diff --git a/automotive02_sorted_single_32.uvl b/automotive02_sorted_single_33.uvl
index 13dcafd..857ffaa 100644
--- a/automotive02_sorted_single_32.uvl
+++ b/automotive02_sorted_single_33.uvl
@@ -1184,6 +1184,7 @@ features
 									F_98CE4D442A1B
 									F_98E2E3C1BFC6
 									F_990B1910A5C9
+									F_99BC61D220BD
 									F_99EBFB0F4B15
 									F_9AC869816CDE
 									F_9CADD809A9F2

