diff --git a/automotive02_sorted_single_84.uvl b/automotive02_sorted_single_85.uvl
index fc39f6c..2e6aea2 100644
--- a/automotive02_sorted_single_84.uvl
+++ b/automotive02_sorted_single_85.uvl
@@ -3324,7 +3324,6 @@ features
 							N_2072278899__F_521A39AAD3B3
 							N_2072278899__F_AB6AB913B139
 							N_2072278899__F_C1D4E9ECB37A
-							N_2072278899__F_C2E08D06CA46
 						optional
 							"F_422E95C1BE08"
 								or

