diff --git a/automotive02_sorted_single_56.uvl b/automotive02_sorted_single_57.uvl
index a6be8aa..8fac557 100644
--- a/automotive02_sorted_single_56.uvl
+++ b/automotive02_sorted_single_57.uvl
@@ -2723,6 +2723,7 @@ features
 									F_01867A96760A
 									F_12B04EDB4EE7
 									F_15CD50CDD382
+									F_1CF3C7013EF0
 									F_2150B391315C
 									F_3F56F4A2079D
 									F_58D808702C34

