diff --git a/automotive02_sorted_single_29.uvl b/automotive02_sorted_single_30.uvl
index 861e3a0..2974c6d 100644
--- a/automotive02_sorted_single_29.uvl
+++ b/automotive02_sorted_single_30.uvl
@@ -966,6 +966,7 @@ features
 											I_189205934__F_2EA209C0B60B
 											I_189205934__F_3412DC316CF1
 											I_189205934__F_49C6AD8077B9
+											I_189205934__F_4A59E95ECB4D
 											I_189205934__F_50C1B9F563B7
 											I_189205934__F_558981D7E2F2
 											I_189205934__F_5EAF2384F11A

