diff --git a/automotive02_sorted_single_83.uvl b/automotive02_sorted_single_84.uvl
index 624989a..fc39f6c 100644
--- a/automotive02_sorted_single_83.uvl
+++ b/automotive02_sorted_single_84.uvl
@@ -3320,7 +3320,6 @@ features
 									F_06A04DAFF09C
 									F_238611C690C3
 							N_2072278899__F_17FDEBA2FE48
-							N_2072278899__F_3EDEC7308306
 							N_2072278899__F_3FA40712058B
 							N_2072278899__F_521A39AAD3B3
 							N_2072278899__F_AB6AB913B139

