diff --git a/financialservices_single_95.uvl b/financialservices_single_96.uvl
index c9d10cc..bd94f5e 100644
--- a/financialservices_single_95.uvl
+++ b/financialservices_single_96.uvl
@@ -1099,6 +1099,8 @@ constraints
 	F_8pVlZ1HPL7qiGe96zxnKyiyZ1p0tjEFM => "F_AtfEDJkBn0QpwIN6Ywe+VmaCDI5oqa/k" & F_AFVaLF1WnY4n7Py8npxgt3y6uyGX6W3R & "F_bUkvh+4c33rD22ISEi1KC87FRD4+8bG4" & "F_goxfwEubWjnoxBi7uXfhjv8+1qdnTx6M" & "F_5KghHv1tg8t+TIA8GKazGdKn/SwU+ims"
 	F_8pVlZ1HPL7qiGe96zxnKyiyZ1p0tjEFM => !"F_tcm3JQ8JP9ls4DV3hxd0XMfmjhhSyWXF"
 	F_8pVlZ1HPL7qiGe96zxnKyiyZ1p0tjEFM => "F_Tt90gxJ6eUlE+kgakSVIcjCeuBuu9InX"
+	F_kHjJv4X6fOHUk8UDJvJ55qKHg33xO7pW => F_MA4Kf1gSwbZTj0EfBY3j0c7gccduijm3 & "F_ykbAki/rQXJa0cKDVZqM4Oe/z1EQ9Uqe" & "F_bUkvh+4c33rD22ISEi1KC87FRD4+8bG4" & "F_goxfwEubWjnoxBi7uXfhjv8+1qdnTx6M" & "F_5KghHv1tg8t+TIA8GKazGdKn/SwU+ims"
+	F_kHjJv4X6fOHUk8UDJvJ55qKHg33xO7pW => !"F_tcm3JQ8JP9ls4DV3hxd0XMfmjhhSyWXF"
 	F_kHjJv4X6fOHUk8UDJvJ55qKHg33xO7pW => "F_Tt90gxJ6eUlE+kgakSVIcjCeuBuu9InX"
 	F_RNKj2KH7pmOEzMCCifwo4kBP5vn2FdPh => "F_Tt90gxJ6eUlE+kgakSVIcjCeuBuu9InX"
 	F_xd5sS9wGOXbilf1VLgv7ikFcZLFjr5lB => "F_+h4YuNgGaf6Y+gGrymDfvsR69sbyA9ue"

