diff --git a/financialservices_single_51.uvl b/financialservices_single_52.uvl
index e68dd7c..4515a29 100644
--- a/financialservices_single_51.uvl
+++ b/financialservices_single_52.uvl
@@ -953,6 +953,7 @@ constraints
 	"F_5XxaCaC2aV55SqR705OPbbaGSw98NGNv" => !"F_tcm3JQ8JP9ls4DV3hxd0XMfmjhhSyWXF"
 	"F_ua0D7Bxzalu7G79zHcSvG+vT5HaD919e" => !"F_XQwgWzAohf7+b7btijunWWILUXVTM17l"
 	"F_ua0D7Bxzalu7G79zHcSvG+vT5HaD919e" => F_PvkPn7tRDyUhER4OxpsndnWaAkzW4lQ4
+	"F_ua0D7Bxzalu7G79zHcSvG+vT5HaD919e" => !"F_tcm3JQ8JP9ls4DV3hxd0XMfmjhhSyWXF"
 	"F_ua0D7Bxzalu7G79zHcSvG+vT5HaD919e" => "F_mbpn+qcK98lH4QCBsim8ds59KIwArsfo"
 	"F_ua0D7Bxzalu7G79zHcSvG+vT5HaD919e" => "F_Tt90gxJ6eUlE+kgakSVIcjCeuBuu9InX"
 	"F_ua0D7Bxzalu7G79zHcSvG+vT5HaD919e" => "F_AxocsKXEl0LyjQ++K9/QkliHzt2xvqOj"

