diff --git a/financialservices_single_12.uvl b/financialservices_single_13.uvl
index 34937c8..adf46c0 100644
--- a/financialservices_single_12.uvl
+++ b/financialservices_single_13.uvl
@@ -630,10 +630,6 @@ features
 									"F_hJNBdoDJ+oNlS7Q1RVtlYQ069nvtxy7V"
 									F_DL413wKdV72o0S4wVyF3blat7zQrZEkK
 									"F_EYDTJEMBR/twt9w+5brkm1Xga7GmOBXi"
-					"F_wLGQBdI/s79pqJg2MF1NpxTZQqQlMPOa"	
-						alternative
-							"F_u1y6OIoZcUi1V+2Xoqp2kTZHosL8xE5A"
-							F_tHNOzSGtslzjFRQzn9LggYyLAlKmGI7I
 					"F_vBr+6gUFW1dGdCV0YxO4b3xilIXlAQy7"	
 						mandatory
 							"F_KXLUem1AGIdaWDaeiEVYMJG5BXauUuXU"	

