diff --git a/financialservices_single_05.uvl b/financialservices_single_06.uvl
index e3cd435..99a5a8c 100644
--- a/financialservices_single_05.uvl
+++ b/financialservices_single_06.uvl
@@ -478,6 +478,70 @@ features
 												alternative
 													F_3pswupNm3u4GdKsWh5UVFZT1WDmt3IGQ
 													F_0Q1cPIMpcsKmgRGPQAoSTFL1BO8VDhT7
+					"F_hxWM93xnptsnhuWx5iEI8COzxcV5sg2F"	
+						mandatory
+							"F_QBdXnIgdKW8OKEQk9zWDhMJNidvEPaHQ"	
+								alternative
+									"F_aWxI07K/R8tp24YqAcBzDxK/ws7Q9lgA"	
+										alternative
+											"F_6qVIyXYSsd0OWb8FKLpZ2J736FNVmzHJ"	
+												mandatory
+													F_kS5fIfzAj3jxTvYn4UiOfXzW66kKjt05
+											"F_pYmwtII6wNHrcWRw/uNopv+aGrZ9KeE/"	
+												mandatory
+													"F_miZh0kR1oa6F8OKTrKj4odh7GJ43lOe/"
+											"F_XVU5LrH34c6w9+w6JzWMSXx24u+8smLI"	
+												mandatory
+													"F_PYBkDxeqqxkn3j2dZ7QBGw0AgYb+hzRb"
+											"F_Cp426hV4SZGCSwSHuk5aYuIe5jmeluPQ"	
+												mandatory
+													"F_961Cmfrg17W5DncOWmqj16+C+SC9VZhw"
+									"F_034CdzwIf4nI97zOGpisQ0IKYeD9qhym"	
+										mandatory
+											"F_kiMPxAxnN1PEDebDPNlSXkvF4/tNZYYj"	
+												mandatory
+													F_sojcJllwoiIL59yIr1n9FHc2FtIYlZtU
+			"F_gkEltCN+VBzJqxo9MlvUSx5R/5jKcIoq"	
+				mandatory
+					"F_KJmyyNTslVaem5ksCY+KYa8DFmvOhkMl"	
+						alternative
+							"F_eqilJeueSQCXGqIq1Jmo+72MmPiMkwwN"
+							"F_b3TjqYwBcWeznPwKeSggVd0/rSovzzPY"
+							"F_/PfxYiY0mCoMZQqEN+dHk/2Siw8Ua0Wg"
+							"F_sCVhH/YpjAhAU5mo5NTYNpzV9U/4CQUN"
+							F_wEdFlRoGc3TVvb4UOSwCikemuVNjQbkt
+							"F_Kd+bOy7/S85V1jkCG1+Tt1hkebem+RT5"	
+								alternative
+									F_9A2Zrd79pBzKt307rV90rbLkScNRhK7p
+									F_0PCk5JpWk7n9H3wwx80ldgfV1UWoMBKp
+									F_7r7TUhopAoyPp8inaAWZ0tqbsmjRJMXN
+									"F_Ry9/xs3fApFO981maU/8kjlnhCDFjcce"	
+										alternative
+											"F_QQTyUkOpHqdC21UK388ci3cZ1BX/GOee"
+											"F_kz2yy7sBnx4lf4i3mQKl1e+PCNaPaWWD"
+							F_EaGYkDUa4LnpOgsa5gPrWByAOsc9zwGF
+							"F_yJiBrgWXNFkB32Dy+6Rut7+AGDvlktPd"
+							"F_/5CSugGS5fdFIdMOWYQXRm/znNRLKXJc"
+					"F_SlZNCAn46uPrI0WOcYGFUisBOqVKxfS4"	
+						alternative
+							"F_59+df9FcKVDmhwfSGgRjm8dlsTVAA0bS"
+							"F_OVMcs3tzEKGIf8I7eOs+m04t0nPwyAKy"
+
+				optional
+					"F_fzYZn51YBaK3QjQSpIz5TzzToy/oeH7V"	
+						mandatory
+							"F_QVkXYOWpMxB/zX1kRjw9GDxmi3mZDkjg"	
+								alternative
+									"F_isML1D/sQQJmOcCAuDbNQv783Dch+PS9"
+									"F_+iyn9GOqB64DKgVJfJJwCJwLqDbg/01m"
+									F_bfYkth5zJ9LLGmptXLqQpb0s43JAp8J0
+									F_GwCkAPXHr6oh6h6Jfcb9pihphEObA8Xr
+							"F_7r8goOEH79aERI3pdVVTQA/s+Rb6flmd"	
+								alternative
+									"F_yHHPfuI7yVn/2nAr7js4D7kdNNwZ9nmw"
+									"F_bVLch/Pg4A94fugjWbrj72oNUZLW1gxX"
+									"F_HU+tnu5lpN6d3ouUJmlx7/2l4WhoQ/WY"
+									"F_UlT+nkZ09V3GlCmXv3sj4/bKmfycxlUX"
 			"F_ptX3ptQ5whQZOadnkC9rjRRGJUnH12TV"	
 				mandatory
 					"F_5s0uBvPqcofo9JWXOQymR3brdwMviRL7"	

