diff --git a/financialservices_single_59.uvl b/financialservices_single_60.uvl
index 8d22da6..40d83bf 100644
--- a/financialservices_single_59.uvl
+++ b/financialservices_single_60.uvl
@@ -983,6 +983,7 @@ constraints
 	"F_vbGIWS4HLmVJJs6UtABShF3u7RXiZV8O" => !"F_tcm3JQ8JP9ls4DV3hxd0XMfmjhhSyWXF"
 	"F_vbGIWS4HLmVJJs6UtABShF3u7RXiZV8O" => "F_9vCxyvFnhfqTFtQJdeyRk31eYEUK/pCo"
 	"F_vbGIWS4HLmVJJs6UtABShF3u7RXiZV8O" => "F_Tt90gxJ6eUlE+kgakSVIcjCeuBuu9InX"
+	"F_izYPCrwOWhIu3wGzEDYHInQLd8U+LxMI" => !"F_lgci9rQfZar3KgcL4lr3rgbiIWUEMbJ2"
 	"F_izYPCrwOWhIu3wGzEDYHInQLd8U+LxMI" => "F_ElWRZDJ9cufYxM/t7tlA1jHyjm5PZko+"
 	"F_rShgqEcFN3onjb12YB/dHngdKS6pYlKY" => !"F_LqaWZKu4g3UG5l6IGHFN0EqLJ7+Kkqu7"
 	"F_rShgqEcFN3onjb12YB/dHngdKS6pYlKY" => "F_AxocsKXEl0LyjQ++K9/QkliHzt2xvqOj"

