diff --git a/automotive02_single_53.uvl b/automotive02_single_54.uvl
index 7e64fdb..c4c3998 100644
--- a/automotive02_single_53.uvl
+++ b/automotive02_single_54.uvl
@@ -3164,6 +3164,12 @@ features
 											F_7B8006020B6C
 
 								optional
+									F_71EF29917042
+									F_B2A97193A557
+									F_24BB5F2908A2
+									F_A09B47656D49
+									F_7AD8D95A7A3C
+									F_2C566929C54D
 									F_06CB64B5BAB4
 							"F_9AF7532651D9"	
 								optional

