diff --git a/financialservices_sorted_single_75.uvl b/financialservices_sorted_single_76.uvl
index ca0d413..74973a6 100644
--- a/financialservices_sorted_single_75.uvl
+++ b/financialservices_sorted_single_76.uvl
@@ -1015,6 +1015,10 @@ constraints
 	"F_aFhQBEMJtWcXQPsXnV3iz76a1oLy/WZ9" => F_CNRzr6IMrzYHsSt3HrP3SNEr8MxFgE9B & F_AFVaLF1WnY4n7Py8npxgt3y6uyGX6W3R & F_5Wt1GLcN9ZXR12Ywk1iK0J4JOrunogoq & F_HLNWVjSIXoeDpAOazTPDvkxxE05mhs8s
 	"F_Agb1T7rPrJ64GsMUvbuVGxy/L2HekK2W" => !"F_tcm3JQ8JP9ls4DV3hxd0XMfmjhhSyWXF"
 	"F_Agb1T7rPrJ64GsMUvbuVGxy/L2HekK2W" => F_EAyJQ2UOspePaKuQVgX5YVFt87jbJC8H
+	"F_Agb1T7rPrJ64GsMUvbuVGxy/L2HekK2W" | "F_1dbzvoSVNLs0szkJRIS1wypFEFxpKum7" & F_wG7mFnoAgFfJVVHRWh0Xqs4fRjHMJnWj => "F_fkf0+YX+AXqZ1OmiHBQhzOPYMR46iLy4" & F_5QtOQgu17AOBnUu5jrLex7loqMjH03iU & "F_VhxSz9M7mHNAxHrQ2ybMks49/YDStEdJ"
+	"F_Agb1T7rPrJ64GsMUvbuVGxy/L2HekK2W" | "F_1dbzvoSVNLs0szkJRIS1wypFEFxpKum7" => F_wG7mFnoAgFfJVVHRWh0Xqs4fRjHMJnWj
+	"F_Agb1T7rPrJ64GsMUvbuVGxy/L2HekK2W" | "F_HyKStxsMGoNAdIIqgQanPs4qBqjB148Y" | "F_qTMCSGRMHLZlzKtfyqtQBUgOy3M/lUpe" | "F_v7za7Q20481MIxnLeVYE0b6WSRMp645W" => "F_L3U2vZXv3330ziDwiLkSolecOW8+X4e1" | F_kahHUaOHRB3noBDeYGxGr49z21twRoWr | "F_IpQYUVuU/6M04KPmgv5w7P6EHsHUgeCw" | F_IKBaCZkhqR873ydn98irb62Uce8AtLov | "F_Mx17yla0jlhjzCABAPe/xgNtZZHSGHcl" | "F_/0KFWRmTdEoNb1jFtvQCHT68JM4EV661" | "F_je5LfPP71HJq07R6/PekW5g1SdaUgHcH" | "F_4U30tJk85NjdKyNNlb0s54c5fQ4j5lJ+" | "F_GB40keNLZqjKhL4HxXOcYq8seB84Bb+F" | "F_XYkRfm6D6FReiwMieP2gA9ZqA1/m0oXs" | "F_3EUEicw4h3/ZMbNVZGjR5Mb74LIe/RGh" | "F_MXipY/+JrRWziuKzbvP3WRjY9F1LY/SI" | F_2ShELK86yaTG37RoxfBEW4s2GmPN6k0Y | "F_FtHMPygjZOi9rhr9UuJadj42A7rWY+RW"
+	"F_ai/ymS8a0lm6wYEEUw7qXvkfapnCxiG3" => "F_L3U2vZXv3330ziDwiLkSolecOW8+X4e1" | F_kahHUaOHRB3noBDeYGxGr49z21twRoWr | "F_IpQYUVuU/6M04KPmgv5w7P6EHsHUgeCw" | F_IKBaCZkhqR873ydn98irb62Uce8AtLov | "F_Mx17yla0jlhjzCABAPe/xgNtZZHSGHcl" | "F_/0KFWRmTdEoNb1jFtvQCHT68JM4EV661" | "F_je5LfPP71HJq07R6/PekW5g1SdaUgHcH" | "F_4U30tJk85NjdKyNNlb0s54c5fQ4j5lJ+" | "F_GB40keNLZqjKhL4HxXOcYq8seB84Bb+F" | "F_XYkRfm6D6FReiwMieP2gA9ZqA1/m0oXs" | "F_3EUEicw4h3/ZMbNVZGjR5Mb74LIe/RGh" | "F_MXipY/+JrRWziuKzbvP3WRjY9F1LY/SI" | F_2ShELK86yaTG37RoxfBEW4s2GmPN6k0Y
 	"F_aoAygbOG9evSs8zs+LBPPokqY5vY4KAe" => "F_+h4YuNgGaf6Y+gGrymDfvsR69sbyA9ue"
 	"F_ApcqB37+NqKVV76DXKGm/UEkxEkEptfD" => "F_Tt90gxJ6eUlE+kgakSVIcjCeuBuu9InX"
 	"F_B1wk5828hO8roE3GFQukYFD/6tuKGJAM" => "F_+h4YuNgGaf6Y+gGrymDfvsR69sbyA9ue"

