diff --git a/financialservices_sorted_single_12.uvl b/financialservices_sorted_single_13.uvl
index 5b8812a..b1641fc 100644
--- a/financialservices_sorted_single_12.uvl
+++ b/financialservices_sorted_single_13.uvl
@@ -225,6 +225,37 @@ features
 							"F_ix63yBeFIEBKnGkpBReQ6afAB9lMg+h3"
 							"F_M8O6ICKdrLVt/XHRlpqXwVP4NTiaO7BE"
 							F_da2U6TULf1cuMBu3u8rwyihSt7jPo05A
+					"F_Iw9llUh7xVq3QBA82MA+LB9gURpscyPk"
+						alternative
+							"F_/0KFWRmTdEoNb1jFtvQCHT68JM4EV661"
+							"F_3EUEicw4h3/ZMbNVZGjR5Mb74LIe/RGh"
+							"F_4U30tJk85NjdKyNNlb0s54c5fQ4j5lJ+"
+							"F_FtHMPygjZOi9rhr9UuJadj42A7rWY+RW"
+							"F_GB40keNLZqjKhL4HxXOcYq8seB84Bb+F"
+							"F_IpQYUVuU/6M04KPmgv5w7P6EHsHUgeCw"
+							"F_je5LfPP71HJq07R6/PekW5g1SdaUgHcH"
+							"F_L3U2vZXv3330ziDwiLkSolecOW8+X4e1"
+							"F_Mx17yla0jlhjzCABAPe/xgNtZZHSGHcl"
+							"F_MXipY/+JrRWziuKzbvP3WRjY9F1LY/SI"
+							"F_XYkRfm6D6FReiwMieP2gA9ZqA1/m0oXs"
+							F_2ShELK86yaTG37RoxfBEW4s2GmPN6k0Y
+							F_cbye7ZYMtMF4AqLKYtEDq3kXjmUpHHqe
+							F_IKBaCZkhqR873ydn98irb62Uce8AtLov
+							F_kahHUaOHRB3noBDeYGxGr49z21twRoWr
+					"F_Izdnce8WHedEc8NAVr+RqD4g6pHQUYdn"
+						alternative
+							"F_u1y6OIoZcUi1V+2Xoqp2kTZHosL8xE5A"
+							F_tHNOzSGtslzjFRQzn9LggYyLAlKmGI7I
+					"F_oaaw/o22DF9f2hYhXM57dB7Rpez1Eetv"
+						alternative
+							"F_XTbSnqsL+ABNwGwZKE3z5B4hnCQ9uPIL"
+							"F_zONxzU86LeQnEQV20h3FCa/12GC7NGzJ"
+							F_97k4wTQ2FHgEkgwutQRJkHRZhoeTni3c
+							F_TahYT1ii6fDx4mYL7R6IzhQGD8MBOZy5
+					"F_YYbH9ZBDxO9HzJUMItSN0Mu6wi8wQ/gj"
+						mandatory
+							"F_u3NLZKSnTX9Z/VjGpL11jvNyjgSw8Z+x"
+				optional
 					"F_K8DbZtFgUSs9wIjTWKjuaMtJ5dexTjVx"
 						mandatory
 							"F_CKQYjMSN2vGfTasaJbzLRsJDnh6QYEqz"

