diff --git a/financialservices_sorted_single_02.uvl b/financialservices_sorted_single_03.uvl
index 18c24bc..d3191f7 100644
--- a/financialservices_sorted_single_02.uvl
+++ b/financialservices_sorted_single_03.uvl
@@ -18,6 +18,7 @@ features
 								alternative
 									"F_/rtxGEYKdSUo7Xzw6VhroBRwm8xPT0kC"
 										alternative
+											"F_9vCxyvFnhfqTFtQJdeyRk31eYEUK/pCo"
 											"F_GoPlX8cMSG9j1vn08abl+MK8EpUW3/1o"
 											"F_u0bcBeripG3FZ0+CUezzAmQlDiurV1Lx"
 											F_gLkc9IBHY9fjNeIL6ubTZWaAfiFo6Tqa

