diff --git a/financialservices_sorted_single_15.uvl b/financialservices_sorted_single_16.uvl
index 7683ba4..ca71475 100644
--- a/financialservices_sorted_single_15.uvl
+++ b/financialservices_sorted_single_16.uvl
@@ -277,6 +277,34 @@ features
 								alternative
 									F_1NsAegideb9PH4VPjX7I4LrF1MLTW642
 									F_jUGvRmWx5Q8dSpgSHLR8KAFcVUXAPy9i
+							"F_RnVFo07OkUmg9nT/G0ZK2jgVGrDqXM/K"
+								mandatory
+									F_5Ddv6bCvJ409Dlrx7WhkPS2AI8tOvPGC
+					"F_lgci9rQfZar3KgcL4lr3rgbiIWUEMbJ2"
+						alternative
+							"F_/R/L83l+nY78xhswOBtyX+cgMEgj5r2j"
+							"F_6xfdwaGi9GVgLs3x1N6CZ72ad/xW3bGJ"
+							"F_FthJSnP5/UyZGhi3OU28PyD3dWbKoF6T"
+							"F_lenFQUglomofciAVwtz+ya6UWyQClVf6"
+							"F_MM2cDp5KnOCDwjDZKnV+8aYdfUqbwA71"
+							F_03GpBxG61jdyj1wkcXQfQxmleOMwyRHu
+							F_xCGioBkRd1DblpqtSOUM3AK8056LpM4a
+							F_xVBo09ujH04BTJGBUs1ENBkeBSr6QnrO
+					"F_tcm3JQ8JP9ls4DV3hxd0XMfmjhhSyWXF"
+						alternative
+							"F_+h4YuNgGaf6Y+gGrymDfvsR69sbyA9ue"
+							"F_0AgBM9+Sn5JrBR7eml4VhFsvc9qKGwam"
+							"F_EikjIDBitX8c/Slm7MFedP5HA57WA9jm"
+							"F_EuC/rKXPKhVy+of/dBH+CH+GxqnZDIRR"
+							"F_geybKOUsZoYzYVPqfL+bf7Ai2uguvxAb"
+							"F_sq/ZooxFsdAeCSzLJJYaghQ3hPcl9zH+"
+							"F_TncVyAii7//qhH1OpE6LkGTGlffz3gf4"
+							"F_TzCWScqOkYMx/tYYBcCQxYu9TrQnFrFe"
+							"F_WB4sLSxgH5mR+BOwok5VyzX+LC3NU3RE"
+							"F_YBj5IYLejKZ7HVSjn1/MKHB0WSSuQfJu"
+							F_bIIpX3EdnkBFOlU6w6OldlZARjJj6Jhi
+							F_KW4JAiyunPk9dmhcs93SwMipxGqjvOY3
+							F_O4qv3pUvic6NXA64barPy3jKkIsB9Cb4
 					"F_vBr+6gUFW1dGdCV0YxO4b3xilIXlAQy7"
 						mandatory
 							"F_KXLUem1AGIdaWDaeiEVYMJG5BXauUuXU"

