diff --git a/financialservices_sorted_single_51.uvl b/financialservices_sorted_single_52.uvl
index d464f8f..d7168ac 100644
--- a/financialservices_sorted_single_51.uvl
+++ b/financialservices_sorted_single_52.uvl
@@ -928,6 +928,8 @@ constraints
 	"F_2GK12Bc3EfubAsjOPiyVgHy/9ZlwRybj" => "F_AtfEDJkBn0QpwIN6Ywe+VmaCDI5oqa/k" & F_AFVaLF1WnY4n7Py8npxgt3y6uyGX6W3R & "F_bUkvh+4c33rD22ISEi1KC87FRD4+8bG4" & "F_goxfwEubWjnoxBi7uXfhjv8+1qdnTx6M" & "F_T0obdGYu+dREPcQ3VWwApkDkW5mIHohm"
 	"F_2GK12Bc3EfubAsjOPiyVgHy/9ZlwRybj" => "F_Tt90gxJ6eUlE+kgakSVIcjCeuBuu9InX"
 	"F_2Oj9hTUS+U+KGdHsYO+y9trg7ROBzGgY" <=> "F_+aaTsPojzvf7RUcbfhTwaD4Xk67YJU0Z" & "F_/AxojXNiG2rXLe6Gnj5XFrrF6xsOBLmx" & "F_5w20X1Mhjfr7N6kc7Jwix+fCwrrW3ptb"
+	"F_2sbTC8jtJofmH3xtT/0GJ8m+y+ggQUD0" => !"F_tcm3JQ8JP9ls4DV3hxd0XMfmjhhSyWXF"
+	"F_2sbTC8jtJofmH3xtT/0GJ8m+y+ggQUD0" => "F_hjPj2V+MoNm6ZyFiRMTQDIe8Sqol1oMQ" & "F_n4p+Vzi2nxqZeITJKltA6qyv124dN2/0" & "F_hK/3d+kJrbl/+ymLl4B2FA8SVnrFQ/ap" & "F_goxfwEubWjnoxBi7uXfhjv8+1qdnTx6M" & "F_T0obdGYu+dREPcQ3VWwApkDkW5mIHohm"
 	"F_2sbTC8jtJofmH3xtT/0GJ8m+y+ggQUD0" => "F_Tt90gxJ6eUlE+kgakSVIcjCeuBuu9InX"
 	"F_2sbTC8jtJofmH3xtT/0GJ8m+y+ggQUD0" => F_da2U6TULf1cuMBu3u8rwyihSt7jPo05A & !"F_XQwgWzAohf7+b7btijunWWILUXVTM17l"
 	"F_2yh9bKLY8rXm/ovIVXZEPmTwFq8UEOq6" => "F_63eeYAr+YiKsgfMlLw6LyVak++XCUx4+" & !"F_XQwgWzAohf7+b7btijunWWILUXVTM17l"

