diff --git a/financialservices_sorted_single_25.uvl b/financialservices_sorted_single_26.uvl
index 1d525cc..9a19106 100644
--- a/financialservices_sorted_single_25.uvl
+++ b/financialservices_sorted_single_26.uvl
@@ -744,6 +744,8 @@ features
 											"F_pToE60L30CHFBZnVAkRBFn1jkuH92SZy"
 												alternative
 													"F_qIQNPCJZXvgVSMjfxbJb0DmpMWR+vruI"
+													"F_xj6zKAjzCqvEKl9/fChWOD+8uM24l9HS"
+													F_9qPPiSMKWqVlvv3AbGeuyudF1Xy9oEDZ
 													F_bIjXpMWwXr3nh2uyOJqpyQpBMRdl4dmz
 													F_QizD1xTV4oABGTc63YT3Hp5JTYRMNQ6D
 													F_UnLk3P6nMHQf2UXsiwkHxeQxCkX7nN2V

