diff --git a/financialservices_sorted_single_31.uvl b/financialservices_sorted_single_32.uvl
index 46701ba..099c436 100644
--- a/financialservices_sorted_single_31.uvl
+++ b/financialservices_sorted_single_32.uvl
@@ -857,7 +857,7 @@ features
 													"F_sWjqPnA8gMytHL6t6exu0+5GE7QVWXWK"
 													F_x6pK46SwpIXYRUuxqlUqIEgkwgY54jtQ
 											"F_yGIoifoS1Pw69ToDll9/uq9pw/DgtUZX"
-												mandatory
+												alternative
 													F_fHziG2zqN1DTsTaUfwOQNhjUXIaFO6Qt
 constraints
 	"F_/+RbSnuGRJQWJ4lH1ZSbfQu4/T72WijW" => "F_Tt90gxJ6eUlE+kgakSVIcjCeuBuu9InX"

