diff --git a/financialservices_sorted_single_24.uvl b/financialservices_sorted_single_25.uvl
index 4cb9c0e..1d525cc 100644
--- a/financialservices_sorted_single_24.uvl
+++ b/financialservices_sorted_single_25.uvl
@@ -743,8 +743,6 @@ features
 													"F_wjC1ANPFh051HBfT3wll/jIEPBpNUx1c"
 											"F_pToE60L30CHFBZnVAkRBFn1jkuH92SZy"
 												alternative
-													"F_+AwzF6MeLXAgKt37Fm68sW95V5uHc6Xh"
-													"F_7tIdLs7+vDQroQMMH0QBEGzu0+082rGn"
 													"F_qIQNPCJZXvgVSMjfxbJb0DmpMWR+vruI"
 													F_bIjXpMWwXr3nh2uyOJqpyQpBMRdl4dmz
 													F_QizD1xTV4oABGTc63YT3Hp5JTYRMNQ6D
@@ -868,7 +866,6 @@ constraints
 	"F_/wOZ/Y0HWGyr/elbj1SeGbGjxQ+sNEih" => "F_Tt90gxJ6eUlE+kgakSVIcjCeuBuu9InX"
 	"F_+4g9sfx+DhNQrbPY+gaSZICdtF9/lX8a" => "F_Tt90gxJ6eUlE+kgakSVIcjCeuBuu9InX"
 	"F_+7ugA9p+bhXrro5UPB+1+6O/Hmlsg5iI" => "F_+h4YuNgGaf6Y+gGrymDfvsR69sbyA9ue"
-	"F_+AwzF6MeLXAgKt37Fm68sW95V5uHc6Xh" => F_tHNOzSGtslzjFRQzn9LggYyLAlKmGI7I
 	"F_+EJ7Pe6XJnymnfzm5g5WFrJgJIrR/Is2" => "F_+h4YuNgGaf6Y+gGrymDfvsR69sbyA9ue"
 	"F_+EJ7Pe6XJnymnfzm5g5WFrJgJIrR/Is2" => "F_AxocsKXEl0LyjQ++K9/QkliHzt2xvqOj"
 	"F_+h4YuNgGaf6Y+gGrymDfvsR69sbyA9ue" => "F_++rt/tsxsF2/HBzKa8mbpS0jd/2H+9Vq"
@@ -918,7 +915,6 @@ constraints
 	"F_7GmgySp/9vYXxrnsHfZRCsMJx8aKPzcZ" => "F_+h4YuNgGaf6Y+gGrymDfvsR69sbyA9ue"
 	"F_7RWpqEchLizqovQ2E+ODFxbYpWvx7tK3" => "F_+h4YuNgGaf6Y+gGrymDfvsR69sbyA9ue"
 	"F_7RWpqEchLizqovQ2E+ODFxbYpWvx7tK3" => "F_63eeYAr+YiKsgfMlLw6LyVak++XCUx4+" & !"F_XQwgWzAohf7+b7btijunWWILUXVTM17l"
-	"F_7tIdLs7+vDQroQMMH0QBEGzu0+082rGn" => F_tHNOzSGtslzjFRQzn9LggYyLAlKmGI7I
 	"F_7WJcQxhapCOueEnM4N+1CdbG4pRywTyz" => "F_XfVpGQVny8/KgdpwuwAozULzdr1TDTnR"
 	"F_82bLhQ4T7EN5Ay2uEeJXedMH5j+hgK4H" => "F_AxocsKXEl0LyjQ++K9/QkliHzt2xvqOj"
 	"F_86Nx55QvoD+tolihq79kLe6WiLUzctfF" => "F_+h4YuNgGaf6Y+gGrymDfvsR69sbyA9ue"

